lib/apivore/validator.rb

```diff
@@ -17,7 +17,12 @@ def initialize(method, path, expected_response_code, params = {})
     def matches?(swagger_checker)
       pre_checks(swagger_checker)
 
-      unless has_errors?
+      if has_errors?
```

Comment:
this only checks is the `pre_checks` give errors, which is the structural stuff, before the request gets made. I'll do some tests, but I think this needs to be moved to the end of the method?